content/04.experiment-types.md

```diff
@@ -16,4 +16,80 @@ Proteomics experiments can be both qualitative and quantitative.
 - Protein stability and small molecule binding (Thermal proteome profiling, TPP, or cellular thermal shift assay, CETSA)
 - Protein-protein interactions (PPIs): AP-MS, APEX, BioID
 
+###Protein abundance changes
 
+### Phosphoproteomics
+Phosphoproteomics is the study of the phosphorylation of proteins which can be used to identify proteins, associated proteins, and their post-translational modifications. Currently, there are three techniques that are useful for phosphoproteomics: mass spectrometry and western blotting. As seen in the figure above, quantitative proteomics over a variety of analytical techniques. Ensure that there is sufficient amount of protein before completing a phosphoproteomics project
+- Mass spectrometry
+One of the ways to use mass spectrometry to detect phosphorylation is by adding a stable isotope in order to see where phosphorylation would happen that can be seen at the proteome level. [@doi:10.1073/pnas.0609836104; @doi:10.1038/msb4100182; @doi:10.4049/jimmunol.176.5.2833; @doi:10.1126/scisignal.2000007]  It is important to understand what methodology you are planning to use before running the experiments.
+Despite advancement in technology of phosphoproteomics, the following challenges still exist: limited sample amounts, highly complex samples, and huge dynamic range [@doi:10.1016/j.copbio.2008.06.006]. Current approaches are limited in their ability to detect phosphoproteins at a resolution of 10-15 kD. Additionally, phosphoproteomic analysis is often time-consuming and requires the use of expensive equipment.
+
+### Glycoproteomics
+The goal of glycoproteomics is to understand the interactions between proteins and glycoproteins and to gain insights into the function and regulation of glycoprotein expression. Understanding protein glycosylation will help us understand numerous biological processes since this is a universal protein modification that can be seen across all domains of life [@doi:10.1038/s43586-022-00128-4; @doi:10.1093/glycob/cww086; @doi:10.1016/j.cell.2006.08.019; @doi:10.1146/annurev-neuro-071714-034019] The glycosylation sites can be glycosylphosphates, N-linked glycosylation sites (asparagine-linked), O-linked glycosylation sites (serine/threonine-linked), or Z-linked glycosylation sites. 
```

Comment:
```suggestion
One PTM gaining interest due to its high occurrence and emerging functional roles is glycosylation. 
Protein glycosylation sites can be glycosylphosphates, N-linked glycosylation sites (asparagine-linked), O-linked glycosylation sites (serine/threonine-linked), or Z-linked glycosylation sites. 
Understanding protein glycosylation will help us understand numerous biological processes since this is a universal protein modification that can be seen across all domains of life [@doi:10.1038/s43586-022-00128-4; @doi:10.1093/glycob/cww086; @doi:10.1016/j.cell.2006.08.019; @doi:10.1146/annurev-neuro-071714-034019].

```